### PR TITLE
fix code block round corner display when horizontal scrollbar appears

### PR DESCRIPTION
**Pull request by dpyzo0o**
overreacted

The behaviour of code block round corners is a bit weird. Also the horizontal scrollbar doesn't look nice. Made some improvements to this problem:

overreacted_1

Hope it looks better!

### CODE REVIEW

**Comment by gaearon** [comment]
Looks neat! Can you also make it scroll with momentum on iOS?

**Comment by LJame** [comment]
nmmm

**Comment by dpyzo0o** [comment]
> Looks neat! Can you also make it scroll with momentum on iOS?

Hi Dan, [iScroll](https://github.com/cubiq/iscroll) might be the way to go. But I have no idea how to integrate with Gatsby since I'm just new to it. Maybe someone else can help?

File: src/utils/theme.css [hunk 1]
```diff
@@ -144,6 +139,9 @@ pre[data-line] {
 }
 
 .gatsby-highlight {
+  margin-bottom: 1.75rem;
+  border-radius: 10px;
+  background: #011627;
```
Comment (yusufff) [inline src/utils/theme.css]:
```suggestion
  background: #011627;
  -webkit-overflow-scrolling: touch;
```
Momentum scroll for iOS

Comment (dpyzo0o) [reply src/utils/theme.css]: Sweet! It works like a charm! Thanks @yusufff

**Comment by DSchau** [comment]
@dpyzo0o would you be interested in tweaking this documentation to use this approach on gatsbyjs.org? e.g. on this page -> https://www.gatsbyjs.org/packages/gatsby-remark-prismjs/

We should keep smart defaults, and if this is a worthwhile change, we'd love to document it so that others get the benefits, as well.

**Comment by gaearon** [comment]
thanks

**Pull request merged**